File: 02. RFID/Film_SparePart_RFID/Library.Database/DTO.cs
using System.Data;

namespace Library.Database
{
    /// <summary>
    /// Passing Data between Library and Interface
    /// </summary>
    public class DTO
    {
        private DataTable _table1;
        private bool _error1;
        private string _errorString1;
        private string _str1;
        private int _int1;

        public DataTable Table
        {
            get { return _table1; }
            set { _table1 = value; }
        }

        /// <summary>
        /// Set Error = true if hit error
        /// </summary>
        public bool Error
        {
            get { return _error1; }
            set { _error1 = value; }
        }

        /// <summary>
        /// Pass Error Message
        /// </summary>
        public string ErrorMessage
        {
            get { return _errorString1; }
            set { _errorString1 = value; }
        }

        public string Str
        {
            get { return _str1; }
            set { _str1 = value; }
        }

        public int Int
        {
            get { return _int1; }
            set { _int1 = value; }
        }
    }
}

File: 02. RFID/Film_SparePart_RFID/Library.Database/DTO.vb
''' <summary>
''' Passing Data between Library and Interface
''' </summary>
Public Class DTO

    Private _table1 As DataTable
    Private _error1 As Boolean
    Private _errorString1 As String
    Private _str1 As String
    Private _int1 As Integer

    Public Property Table As DataTable
        Get
            Return _table1
        End Get
        Set(value As DataTable)
            _table1 = value
        End Set
    End Property

    ''' <summary>
    ''' Set Error = true if hit error
    ''' </summary>
    ''' <returns></returns>
    Public Property [Error] As Boolean
        Get
            Return _error1
        End Get
        Set(value As Boolean)
            _error1 = value
        End Set
    End Property

    ''' <summary>
    ''' Pass Error Message 
    ''' </summary>
    ''' <returns></returns>
    Public Property ErrorMessage As String
        Get
            Return _errorString1
        End Get
        Set(value As String)
            _errorString1 = value
        End Set
    End Property

    Public Property Str As String
        Get
            Return _str1
        End Get
        Set(value As String)
            _str1 = value
        End Set
    End Property

    Public Property Int As Integer
        Get
            Return _int1
        End Get
        Set(value As Integer)
            _int1 = value
        End Set
    End Property

End Class
